packages/pnpm/src/cmd/outdated.ts

```diff
@@ -88,42 +91,72 @@ export default async function (
 
   if (!outdatedPackages.length) return
 
-  let columnNames = [
-    'Package',
-    'Current',
-    'Latest'
-  ]
+  if (opts.table !== false) {
```

Comment:
why not just `if (opts.table)` ?

Reply:
I think that might be because it defaults to `undefined`, and `--no-table` gets parsed as `--table=false`.

Reply:
@ExE-Boss I was actually also working on an implementation of this (didn't check if anybody else was working on it 🤦‍♀️) and I also went with `--list` for this reason.

Though I do feel that `--no-table` is much clearer. Also, there is already a `pnpm list` command which might confuse people.

Could we maybe default `opts.table` to `true` unless `--no-table` is explicitly passed?

Reply:
@ExE-Boss You might as well finish up this feature, I don't have a lot more time to spend on it right now.